app/controllers/concerns/phone_confirmation_flow.rb

```diff
@@ -117,7 +116,9 @@ def sms_enabled?
 
   def current_otp_method
     query_method = params[:otp_method]
-    query_method.to_sym if
-      %w(sms voice totp).include? query_method
+
+    return :sms unless %w(sms voice totp).include? query_method
```

Comment:
This change is needed in order for the new controller code to always receive a valid Active Job class. In a subsequent PR, ensuring this will be accomplished via a Form Object with validations that will make sure only accepted delivery methods are submitted.